### PR TITLE
please work

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: trash-dash/objects/obj_gameLine/Create_0.gml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ///@description Game line height and movement
 // You can write your code in this editor
-path_start(path_truck_movement, 1, path_action_continue, false)
-show_debug_message(display_get_height())
\ No newline at end of file
+//path_start(path_truck_movement, 1, path_action_continue, false)
+//show_debug_message(display_get_height())
```
Comment (johnwatson484) [inline trash-dash/objects/obj_gameLine/Create_0.gml]: remove comments

File: trash-dash/objects/obj_mad_dog/Collision_obj_person.gml [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
-global.alive = false
\ No newline at end of file
+
+if(instance_place(x, y, obj_person))
+{
+	global.dog_bite = true;
+}
+
+//global.gameOverMessage = "You were eaten by the dog"
+//global.alive = false
```
Comment (johnwatson484) [inline trash-dash/objects/obj_mad_dog/Collision_obj_person.gml]: remove comment

File: trash-dash/objects/obj_mad_dog/Create_0.gml [hunk 3]
```diff
@@ -1,3 +1,5 @@
 /// @description Insert description here
 // You can write your code in this editor
-audio_play_sound(woof,1,2)
\ No newline at end of file
+global.dog_bite = false
+chase_speed = 3
+//audio_play_sound(woof,1,2)
```
Comment (johnwatson484) [inline trash-dash/objects/obj_mad_dog/Create_0.gml]: remove comment

**Review by FayToward — APPROVED** [review]
LGTM

**Pull request merged**